cli_tools/gce_ovf_import/ovf_importer/ovf_importer.go

```diff
@@ -127,6 +127,9 @@ func (oi *OVFImporter) buildDaisyVars(bootDiskImageURI string, machineType strin
 	varMap["boot_disk_image_uri"] = bootDiskImageURI
 	if oi.params.IsInstanceImport() {
 		varMap["instance_name"] = oi.params.InstanceNames
+		if oi.params.ComputeServiceAccount != "" {
+			varMap["compute_service_account"] = oi.params.ComputeServiceAccount
+		}
```

Comment:
It think you'll want the var for GMI import as well:

daisy_workflows/ovf_import/create_gmi.wf.json

(Unfortunately there's duplication between the two :/ )

Reply:
`create_gmi.wf.json` only creates a GMI from a running instance and there's no way to specify service account in this case. I'm not even sure if GMIs preserve this information. If they do, service account info will be inherited from the source instance.

Reply:
Actually, you're right, `create_gmi.wf.json` also creates the instance. It looks like that, even without setting service account, things work, since no operation is performed on this temporary VM, but I'll update it to have the custom service account in case it gets preserved in GMI.